docs/site-example/site.conf

```diff
@@ -211,6 +211,11 @@
     -- owner = {
       -- obligatory = true
     -- },
+    -- Show/hide the password login
+    -- remote_login = {
+      -- show_password_form = true,
+      -- min_password_length = 12
+    -- }
```

Comment:
Please terminate all lines using a comma, as it will reduce copy-and-paste induced syntax errors.